yaus_c4gt_frontend/src/pages/FormWizard.js

```diff
@@ -0,0 +1,654 @@
+    headers: {
+        authorization: 'authorization-text',
+    },
+    onChange(info) {
+        if (info.file.status !== 'uploading') {
+            console.log(info.file, info.fileList);
+        }
+        if (info.file.status === 'done') {
+            message.success(`${info.file.name} file uploaded successfully`);
+        } else if (info.file.status === 'error') {
+            message.error(`${info.file.name} file upload failed.`);
+        }
+    },
+};
+
+
+const FormDemo = () => {
+
+    const baseUrl = " https://yaus.xyz";
+    const onFinish = (values) => {
+        console.log('Received values of form: ', values);
+    };
+    const [formLayout, setFormLayout] = useState('vertical');
+    const [selectedMenuItem, setSelectedMenuItem] = useState('useUrl');
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    const [modal1Visible, setModal1Visible] = useState(false);
+
+    const onFormLayoutChange = ({ layout }) => {
+        setFormLayout(layout);
+    };
+    const showModal = () => {
+        setIsModalVisible(true);
+    };
+
+    const handleOk = () => {
+        setIsModalVisible(false);
+    };
+    const handleCancel = () => {
+        setIsModalVisible(false);
+    };
+    const history = useHistory();
+
+    const routeChange = () => {
+        let path = `/dashboard`;
+        history.push(path);
+    }
+    const selectBefore = (
+        <Select defaultValue="Default" className="select-before">
+            <Option value="WebUrl">Web Url</Option>
+            <Option value="DeepView">Deep View</Option>
+            <Option value="Defaultt">Default</Option>
+        </Select>
+    );
+    const selectBeforeLink = (
+        <Select defaultValue="Deeplink_path" className="select-before">
+            <Option value="Deeplink_path">$deeplink_path</Option>
+            <Option value="ios">$ios_deeplink_path</Option>
+            <Option value="amndroid">$android_deeplink_path</Option>
+            <Option value="desktop">$desktop_deeplink_path</Option>
+        </Select>
+    );
+
+    const onChange = (value) => {
+        console.log(`selected ${value}`);
+    };
+
+    const [state, setState] = useState({
+        userID: '0fe6ff38-fc46-11ec-b939-0242ac120001',
+        url: '',
+        project: '0fe6ff38-fc46-11ec-b939-0242ac120002',
+        customHashId: nanoid(6),
+        titleImage: '',
+        urlImg: '',
+        description: '',
+    });
+    const handleSubmit = (e) => {
+        const userData = {
+            customHashId: state.customHashId,
+            project: state.project,
+            url: state.url,
+            userID: state.userID
+        };
+        const headers = {
+            'Content-Type': 'application/json',
+            'Authorization': 'JWT fefege...',
+            "Access-Control-Allow-Origin": "*"
+        }
+        console.log(userData);  
+        axios.post(`${baseUrl}/register`, userData, { headers: headers }).then((response) => {
```

Comment:
At some places we are using `axios` and at some places `fetch` is used, be consistent with the method